sources/packages/api/src/database/entities/institution.model.ts

```diff
@@ -78,9 +87,13 @@ export class Institution extends RecordDataModel {
   })
   users: InstitutionUser[];
 
-  @Column({
+  @RelationId((institution: Institution) => institution.institutionType)
+  institutionTypeId: number;
+
+  @ManyToOne(() => InstitutionType, { eager: false, cascade: true })
```

Comment:
For this particular case, I believe that we should set the cascade to false.
The Institution Types are a fixed list that will not be changed normally and, if they changed, should not be in this way.